### ISSUE TITLE
Issue with Zenodo citations

### ISSUE THREAD

**Issue by santisoler** [fatiando/community#145]
The Fatiando packages that don't have a paper associated with instruct users to cite them using the Zenodo entry. The BibTeX entries that Zenodo creates look like this (the one for Choclo):

```
@software{fatiando_a_terra_project_2023_7931023,
  author       = {Fatiando a Terra Project and
                  Soler, Santiago R. and
                  Uieda, Leonardo},
  title        = {{Choclo v0.1.0: Kernel functions for your 
                   geophysical models}},
  month        = may,
  year         = 2023,
  publisher    = {Zenodo},
  version      = {v0.1.0},
  doi          = {10.5281/zenodo.7931023},
  url          = {https://doi.org/10.5281/zenodo.7931023}
}
```

When these entries are rendered in LaTeX they end up looking like this:

- Inline citation: `(a Terra Project et al., 2023)`
- Citation in References: 
  ```
  F. a Terra Project, S. R. Soler, and L. Uieda. Choclo v0.1.0: Kernel functions for your
  geophysical models, May 2023. URL https://doi.org/10.5281/zenodo.7931023.
  ```

The "Fatiando a Terra Project" is treated as first name + middle names + last name, instead of the name of an organization. If we put it inside curly braces, we fix this issue:

```
@software{fatiando_a_terra_project_2023_7931023,
  author       = {{Fatiando a Terra Project} and
                  Soler, Santiago R. and
                  Uieda, Leonardo},
  title        = {{Choclo v0.1.0: Kernel functions for your 
                   geophysical models}},
  month        = may,
  year         = 2023,
  publisher    = {Zenodo},
  version      = {v0.1.0},
  doi          = {10.5281/zenodo.7931023},
  url          = {https://doi.org/10.5281/zenodo.7931023}
}
```

- Inline citation: `(Fatiando a Terra Project et al., 2023)`
- Citation in References:
  ```
  Fatiando a Terra Project, S. R. Soler, and L. Uieda. Choclo v0.1.0: Kernel functions for your
  geophysical models, May 2023. URL https://doi.org/10.5281/zenodo.7931023.
  ```

## Question

Should we include a BibTeX snippet in the Citation pages of our packages? This would solve the current issue by providing our users a quick way to cite our packages without this problem.

Also, I think this could probably worth contacting Zenodo to provide a way to mark authors as organizations instead of real persons.

**Comment by santisoler** [comment]
I'm cc-ing @lheagy here because she faced this issue when trying to cite Choclo.

**Comment by leouieda** [comment]
I agree that we should add those to the citation sections. Makes things easier for people who can copy and paste. I'd also add an AGU style reference on top of that.

Zenodo already introduced the option to have organization authors with their migration to a new system. I've already switched all of our packages to have the first author be a "research group" which was the closest option I could find. The problem is with Zenodo's bibtex export. Using a tool like https://www.doi2bib.org/bib/10.5281/zenodo.7851747 gives the correct bibtex.

**Comment by leouieda** [comment]
Using https://citation.crosscite.org also gives the right format. So the metadata are ok.

**Comment by santisoler** [comment]
Right, Zenodo already knows that "Fatiando a Terra Project" is an organization author. They should probably fix their bibtex export then. That would worth contacting them.

So, if we move forward with this, I think we should add the citations to the doc pages of the packages that don't have an associated paper. And we should also update the release checklist, to add an item reminding maintainers to update the citation bits (with new doi, update authors, etc). Shall we open a new issue for it in this repo?

**Comment by leouieda** [comment]
> And we should also update the release checklist, to add an item reminding maintainers to update the citation bits (with new doi, update authors, etc).

I guess. Maybe we should add the Zenodo citation information to all packages as a way to cite the current version of the package. But make it clear that people should cite the papers as well.

**Comment by santisoler** [comment]
Agreed. If I remember correctly, we used to have something like this in the past haha
